src/rules/__tests__/require-hook.test.ts

```diff
@@ -152,6 +152,18 @@ ruleTester.run('require-hook', rule, {
         });
       });
     `,
+    {
+      code: dedent`
```

Comment:
Could you fix up the indenting on these please? the code itself should be indented one level, and the closing backtick should be at the same level as the properties, e.g.

```
{
  code: dedent`
    enableAutoDestroy(forEach);

    describe('some tests', () => {
      it('is false', () => {
        expect(true).toBe(true);
      });
    });
  `,
  options: [{ excludedFunctions: ['enableAutoDestroy'] }],
}
```